benchmarks/setup.py

```diff
@@ -2,9 +2,19 @@
 
 This package is generally not needed by users of garage.
 """
+import os
 
 from setuptools import find_packages, setup
 
+GARAGE_GH_TOKEN = os.environ.get('GARAGE_GH_TOKEN') or 'git'
+
+REQUIRED = [
+    # Please keep alphabetized
+    'baselines @ https://{}@api.github.com/repos/openai/baselines/tarball/ea25b9e8b234e6ee1bca43083f8f3cf974143998'.format(GARAGE_GH_TOKEN),  # noqa: E501
+    'matplotlib',
```

Comment:
i notice you removed several deps -- are these no longer needed in the benchmark scripts?

Reply:
I checked the codebase and they are not used anywhere.